Package.swift

```diff
@@ -16,21 +16,14 @@ if shouldIncludeDocCPlugin {
     dependencies.append(.package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0"))
 }
 
-let swiftSettings: [SwiftSetting]
-
-#if swift(>=5.9)
-swiftSettings = [ .define("VISION_OS", .when(platforms: [.custom("visionOS")])) ]
-#else
-swiftSettings = []
-#endif
-
 let package = Package(
     name: "RevenueCat",
     platforms: [
         .macOS(.v10_13),
         .watchOS("6.2"),
         .tvOS(.v11),
-        .iOS(.v11)
+        .iOS(.v11),
+        .visionOS(.v1)
```

Comment:
An advantage of this is being able to define this too, even though for some reason it's not actually required.